cpp/src/cider/exec/module/batch/CiderArrowBufferHolder.cpp

```diff
@@ -63,7 +63,6 @@ void CiderArrowArrayBufferHolder::allocBuffer(size_t index, size_t bytes) {
     }
   } else {
     buffers_[index] = allocator_->allocate(bytes);
-    memset(buffers_[index], 0, bytes);  // set new initialized bytes to 0
```

Comment:
I think it's more safe to keep memset here. any performance gain if removed this?

Reply:
about 50% was reduced from `allocBuffer`, which is about 27% in `query_func`.